src/sdam/topology_description.ts

```diff
@@ -342,6 +342,10 @@ export class TopologyDescription {
   hasServer(address: string): boolean {
     return this.servers.has(address);
   }
+
+  toJSON() {
```

Comment:
Can `EJSON.serialize` throw? To my knowledge, it `EJSON.serialize`, uses `EJSON.stringify` then reparses it, and stringify throws on a number of occasions.

Reply:
Yes, EJSON.serialize() can throw.

Reply:
Do we want to add in a try catch here, with a default result, or a doc comment that this function might throw?

Reply:
I can add a doc comment if you'd think that's helpful.  This API isn't intended for direct consumption (although a user could call `TopologyDescription.toJSON()` since it's public).  `toJSON` hooks are called automatically by JSON.stringify(), so it's primarily for use with something like `JSON.stringify(description)`.

I think it's alright as-is.  JSON.stringify() can throw, so anyone calling JSON.stringify() on an object they don't control should be try-catching it.

Reply:
I'd prefer to add a doc comment since it's technically a use case, but also I'm up to ask the team if you want more input!

Reply:
I did add a doc comment but I also looked more closely at the TopologyDescription.  There's nothing that `toJSON()` serializes that would throw when passed to EJSON.serialize() so I marked the function as not throwing in the doc comment.  How does that look?